### PR TITLE
Card

### PR DESCRIPTION
**Pull request by Bibin-Antony**
image

### CODE REVIEW

File: Container/CSS/card.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+body{
+    display: flex;
+    gap: 1rem;
+    flex-wrap: wrap;
+}
+
+.card{
+    display: flex;
+    flex-direction: column;
+    background: #191938;
```
Comment (ishitakeshawani) [inline Container/CSS/card.css]: Please use variables to define and use color

File: Container/CSS/card.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    box-shadow: 0 2px 3px  4px var(--alt-text-color);
+    margin: 1rem;
+    width: 17.75rem;
+    outline: 1px solid var(--alt-text-color);
+    border-radius: 1rem;
+}
+.horizontal{
+    display: flex;
+    flex-direction: row;
+    background: #191938;
```
Comment (ishitakeshawani) [inline Container/CSS/card.css]: Please use variables to define and use color

File: Container/CSS/card.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+}
+.details-content{
+    border-top: 2px solid var(--text-color);
+    font-size: 0.875rem;
+    margin: 0.3rem 0.8rem;
+    padding: 0.8rem 0rem;
+    color: var(--text-color);
+    border-bottom: 2px solid var(--text-color);
+}
+.details-contents{
```
Comment (ishitakeshawani) [inline Container/CSS/card.css:69]: Here you can create a class that has the same properties with details-content and reuse it.
and just need to define other 

classes that have different properties from details-content.

**Review by ishitakeshawani — CHANGES_REQUESTED** [review]

**Comment by ishitakeshawani** [comment]
You can add hover effect on the buy and share button.

**Comment by ishitakeshawani** [comment]
The cards are looking good.

**Comment by ishitakeshawani** [comment]
It will be great if you host it on netlify. So everyone can watch your work in progress.

File: Container/CSS/card.css (hunk 2 above)
Comment (Bibin-Antony) [reply Container/CSS/card.css]: done.

File: Container/CSS/card.css (hunk 1 above)
Comment (Bibin-Antony) [reply Container/CSS/card.css]: done

File: Container/CSS/card.css (hunk 3 above)
Comment (Bibin-Antony) [reply Container/CSS/card.css:69]: my class names are already too huge in the div, ill see what can be done

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good to me. Sharing from what i learned from my reviews  - You can actually include the font weight's in your style.css as variables and also try to use a single unit (preferably rem) in our stylesheets. A class can be created for hover in style.css so that it can be used for all the components.

**Comment by Bibin-Antony** [comment]
changes are made to the card.css

**Pull request merged**